pkg/interactors/create_user_test.go

```diff
@@ -0,0 +1,80 @@
+package interactors
+
+import (
+	"testing"
+
+	"github.com/CESARBR/knot-babeltower/pkg/entities"
+)
+
+type FakeCreateUserLogger struct {
+}
+
+type FakeUserProxy struct {
+	returnValue error
+}
+
+type ErrorMock struct {
+	msg string
+}
+
+func (fl *FakeCreateUserLogger) Info(...interface{}) {}
+
+func (fl *FakeCreateUserLogger) Infof(string, ...interface{}) {}
+
+func (fl *FakeCreateUserLogger) Debug(...interface{}) {}
+
+func (fl *FakeCreateUserLogger) Warn(...interface{}) {}
+
+func (fl *FakeCreateUserLogger) Error(...interface{}) {}
+
+func (fl *FakeCreateUserLogger) Errorf(string, ...interface{}) {}
+
+func (fup *FakeUserProxy) SendCreateUser(user entities.User) (err error) {
+	return fup.returnValue
+}
+
+func (em ErrorMock) Error() string {
+	return em.msg
+}
+
+func TestCreateUser(t *testing.T) {
+	testCases := []struct {
+		name          string
+		email         string
+		password      string
+		fakeLogger    *FakeCreateUserLogger
+		fakeUserProxy *FakeUserProxy
+		expected      string
+	}{
+		{
+			"shouldReturnNoError",
```

Comment:
I don't know if this message is descriptive. What we are testing here? For example, it'd simple if we have the **context** and the **test condition**: "should raise error when the proxy failed to create a user".

There are some patterns (BDD is one of them) but I don't know which one the golang community adopts. Could you take a look?

Reply:
Done